lib/ansible/playbook/included_file.py

```diff
@@ -113,7 +124,15 @@ def process_include_results(results, iterator, loader, variable_manager):
                                     if isinstance(parent_include, IncludeRole):
                                         parent_include_dir = parent_include._role_path
                                     else:
-                                        parent_include_dir = os.path.dirname(templar.template(parent_include.args.get('_raw_params')))
+                                        try:
+                                            parent_include_dir = os.path.dirname(templar.template(parent_include.args.get('_raw_params')))
+                                        except AnsibleError as e:
+                                            parent_include_dir = ''
+                                            display.warning(
+                                                'Templating the path of the parent %s failed. The path to the '
+                                                'included file may not be found. '
```

Comment:
Looks good, shipit. It seems like you eliminated the file not found errors by fixing the search paths. Is there a case where this sentence suggesting that it still may be the problem is needed?

Reply:
To be honest I am not sure, but since we just swallowed the error before, I figured it would be a good idea to at least report any potential error here.